### PR TITLE
Pulled menu for Doordash

### PR DESCRIPTION
**Pull request by SeaWalks**
Also fixed type in detail.mjs for ``parseGrubhub``

### CODE REVIEW

**Comment by SeaWalks** [comment]
image
Sample output for Doordash getStore request in insomnia

**Comment by SeaWalks** [comment]
image
Sample output from insomnia. Note: this will require `puppeteer` and `cheerio` to be installed, which can be done using `npm install puppeteer` and `npm install cheerio`

File: services/Doordash.mjs [hunk 1]
```diff
@@ -188,5 +188,35 @@ class Doordash extends Service {
       throw new HTTPResponseError(res);
     }
   }
+  async getStore(storeID) {
```
Comment (AlfredoSequeida) [inline services/Doordash.mjs:191]: Lets add some documentation to this comment with a description of what it does and @params for the type of expected input data as well as @return so we know what is returned.

**Review by AlfredoSequeida — CHANGES_REQUESTED** [review]
Thanks for the work! I just have one request, after that we should be good to merge!

File: services/Doordash.mjs (hunk 1 above)
Comment (SeaWalks) [reply services/Doordash.mjs:197]: added!

**Review by AlfredoSequeida — APPROVED** [review]
Looks good, thank you for the quick changes!

**Review by dani5612 — APPROVED** [review]
If Alfredo approved, who am I to disapprove?

**Pull request merged**